File: fizz-buzz/FizzBuzz.java
import java.util.List;
import java.util.ArrayList;

class FizzBuzz {
    static List<String> fizzBuzz(int n) {
        List<String> returnList = new ArrayList<>();
        for (int i = 1; i <= n; i++){
            if (i % 15 == 0) {
                returnList.add("FizzBuzz");
            } else if (i % 5 == 0) {
                returnList.add("Buzz");
            } else if (i % 3 == 0) {
                returnList.add("Fizz");
            } else {
                returnList.add(String.valueOf(i));
            }
        }
        return returnList;
    }

    public static void main(String[] args) {
        fizzBuzz(100).forEach(string -> System.out.println(string));
    }
}

File: fizz-buzz/fizz_buzz.py
def fizz_buzz(n: int) -> list[str]:
    say = list()
    for number in range(1, n+1):
        if number % 15 == 0:
            say.append('FizzBuzz')
        elif number % 5 == 0:
            say.append('Buzz')
        elif number % 3 == 0:
            say.append('Fizz')
        else:
            say.append(str(number))
    return say

if __name__  == '__main__':
    print('\n'.join(fizz_buzz(100)))
